search/includes/classes/commands/class-healthcommand.php

```diff
@@ -3,9 +3,9 @@
 namespace Automattic\VIP\Search\Commands;
 
 use \WP_CLI;
-use \WP_CLI\Utils;
 use WP_Error;
 
+require_once __DIR__ . '/../../../../vip-helpers/vip-wp-cli.php';
```

Comment:
How come we're requiring this file now?

Reply:
Please check this: https://github.com/Automattic/vip-go-mu-plugins/actions/runs/4358243887/jobs/7619353489#step:5:382

```
Error: Class 'WPCOM_VIP_CLI_Command' not found in /home/runner/work/vip-go-mu-plugins/vip-go-mu-plugins/search/includes/classes/commands/class-healthcommand.php:16
Stack trace:
#0 /home/runner/work/vip-go-mu-plugins/vip-go-mu-plugins/search/includes/classes/class-search.php(354): require_once()
#1 /home/runner/work/vip-go-mu-plugins/vip-go-mu-plugins/search/includes/classes/class-search.php(209): Automattic\VIP\Search\Search->load_dependencies()
#2 /home/runner/work/vip-go-mu-plugins/vip-go-mu-plugins/tests/search/includes/classes/test-class-settingshealthjob.php(20): Automattic\VIP\Search\Search->init()
```

It looks like it is possible that in some situations the class is not loaded, and it is safer to `require` the helper file explicitly. I wish we were using an autoloader :-(

Reply:
Done.